starkinfra/individualdocument/__individualdocument.py

```diff
@@ -0,0 +1,122 @@
+from ..utils import rest
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+class IndividualDocument(Resource):
+    """# IndividualDocument object
+    IndividualDocuments is an images part of document faces (front face, back face)
+    and self match.
+    When you initialize a IndividualDocument, the entity will not be automatically
+    created in the Stark Infra API. The 'create' function sends the objects
+    to the Stark Infra API and returns the list of created objects.
+    ## Parameters (required):
+    - type [string]: it's a type of IndividualDocument. ex: "cnh-front", "cnh-back", "rg-front", "rg-back" or "selfie"
```

Comment:
`type [string]: type of the IndividualDocument. Options: "cnh-front", "cnh-back", "rg-front", "rg-back" or "selfie"`
São essas todas as opções disponíveis? É sempre bom listar todas